tests/integration/Storefront/Theme/ThemeTest.php

```diff
@@ -1033,4 +1028,16 @@ private function getCustomConfigMultiSelect(): array
             ],
         ];
     }
+
+    private function toKebabCase(string $themeName): string
```

Comment:
Why not make it static in the original class, no need for this duplication

Reply:
Not used anymore due to `XmlParserUtils::kebabCaseToCamelCase($)`